www/include/core/login/login.php

```diff
@@ -46,6 +46,15 @@
 $DBRESULT = $pearDB->query("SELECT `value` FROM `informations` WHERE `key` = 'version' LIMIT 1");
 $release = $DBRESULT->fetchRow();
 
+/**
+ * Getting Keycloak login state
+ */
+$DBRESULT = $pearDB->query("SELECT `value` FROM `options` WHERE `key` = 'keycloak_enable' LIMIT 1");
+$keycloakEnabled = $DBRESULT->fetchRow()["value"];
+
+$DBRESULT = $pearDB->query("SELECT `value` FROM `options` WHERE `key` = 'keycloak_mode' LIMIT 1");
+$keycloakMode = $DBRESULT->fetchRow()["value"];
```

Comment:
what is $keycloakMode for ? -> login.ihtml +26
is it used for the mixed authentication ?